packages/components/src/custom-gradient-bar/control-points.js

```diff
@@ -89,7 +89,7 @@ function GradientColorPickerDropdown( {
 			result.position = 'bottom left';
 		}
 		return result;
-	}, [ gradientPickerDomRef.current, isRenderedInSidebar ] );
+	}, [ gradientPickerDomRef, isRenderedInSidebar ] );
```

Comment:
[Refs should not be added to dependency](https://epicreact.dev/why-you-shouldnt-put-refs-in-a-dependency-array/) arrays — I wonder if ESLint can't pick that up, since it's just an argument of the component function (and is not typed as a ref) ?

Reply:
Fascinating, thank you again for sharing this article. It'll be interesting to see what happens when we type this one!

Reading the above article, I agree with @mirka that it should be safe (albeit unnecessary) to keep it in the array. That also prevents ESLint from throwing a warning that `gradientPickerDomRef` is missing, since it can't yet tell what it actually is.

Reply:
Agreed, let's keep it for now. Hopefully, when we type this component, ESLint will flag it as an unnecessary dependency :)